traceapp/vis.go

```diff
@@ -13,6 +14,8 @@ import (
 	_ "sourcegraph.com/sourcegraph/appdash/sqltrace"
 )
 
+var ErrTimelineItemValidation = errors.New("timeline item validation error")
```

Comment:
Good catch! I don't think it does either. @chris-ramon can you send a follow-up PR for this?